### PR TITLE
vendor: github.com/moby/buildkit 6bd81372ad6f (master)

### PR DESCRIPTION
**Pull request by thaJeztah**
### vendor: github.com/containerd/containerd v1.7.12

full diff: https://github.com/containerd/containerd/compare/v1.7.11...v1.7.12

### vendor: github.com/google/go-cmp v0.6.0

full diff: https://github.com/google/go-cmp/compare/v0.5.9...v0.6.0

### vendor: github.com/google/uuid v1.5.0

full diff: https://github.com/google/uuid/compare/v1.3.1...v1.5.0

### vendor: golang.org/x/sys v0.16.0

full diff: https://github.com/golang/sys/compare/v0.15.0...v0.16.0

### vendor: golang.org/x/tools v0.14.0, golang.org/x/mod v0.13.0, golang.org/x/sync v0.4.0

full diff:

- https://github.com/golang/tools/compare/v0.10.0...v0.14.0
- https://github.com/golang/mod/compare/v0.11.0...v0.13.0
- https://github.com/golang/sync/compare/v0.3.0...v0.4.0

### vendor: github.com/moby/sys/mountinfo v0.7.1

full diff: https://github.com/moby/sys/compare/mountinfo/v0.6.2...mountinfo/v0.7.1

### vendor: github.com/docker/docker v25.0.1

- pkg/system: return even richer xattr errors

full diff: https://github.com/moby/moby/compare/v25.0.0-rc.1...v25.0.1

### vendor: github.com/docker/cli v25.0.1

- cli-plugins: move socket code into common package
- cli-plugins: don't use abstract sockets on macOS
    - fixes CLI leaving behind plugin socket mount-points
- socket: return from loop after EOF

full diff: https://github.com/docker/cli/compare/v25.0.0-rc.1...v25.0.1

### vendor: github.com/moby/buildkit 6bd81372ad6f (master)

- tests: implement NetNSDetached method

full diff: https://github.com/moby/buildkit/compare/6e200afad512...6bd81372ad6f27a8f3bf530e8cd91ea811f1f993

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -7,15 +7,15 @@ require (
...
 	github.com/containerd/console v1.0.3
-	github.com/containerd/containerd v1.7.11
+	github.com/containerd/containerd v1.7.12
 	github.com/containerd/continuity v0.4.2
 	github.com/containerd/log v0.1.0
 	github.com/containerd/typeurl/v2 v2.1.1
 	github.com/creack/pty v1.1.18
 	github.com/distribution/reference v0.5.0
-	github.com/docker/cli v25.0.0-rc.1+incompatible
+	github.com/docker/cli v25.0.0-rc.2+incompatible
```
Comment (thaJeztah) [inline go.mod:16]: - Still rc2; will be updated to rc3 when vendoring after https://github.com/moby/buildkit/pull/4565

Comment (thaJeztah) [reply go.mod:16]: PR above was merged; let me update this PR

**Comment by thaJeztah** [comment]
Thanks @crazy-max ! I thought I'd update to latest (with the "non-rc" version of docker), but didn't check for other changes; do you want me to squash your commit with the buildkit-bump commit?

**Comment by crazy-max** [comment]
> Thanks @crazy-max ! I thought I'd update to latest (with the "non-rc" version of docker), but didn't check for other changes; do you want me to squash your commit with the buildkit-bump commit?

Yes please do

**Comment by thaJeztah** [comment]
@crazy-max rebased, squashed your commit (added you as co-author) and updated to current master, which added https://github.com/moby/buildkit/pull/4587

File: go.mod [hunk 2]
```diff
@@ -7,25 +7,25 @@ require (
...
 	github.com/containerd/console v1.0.3
-	github.com/containerd/containerd v1.7.11
+	github.com/containerd/containerd v1.7.12
 	github.com/containerd/continuity v0.4.2
 	github.com/containerd/log v0.1.0
 	github.com/containerd/typeurl/v2 v2.1.1
 	github.com/creack/pty v1.1.18
 	github.com/distribution/reference v0.5.0
-	github.com/docker/cli v25.0.0-rc.1+incompatible
+	github.com/docker/cli v25.0.0+incompatible
```
Comment (crazy-max) [inline go.mod:16]: Do we want 25.0.1 bump on buildkit first and revendor here?

File: go.mod [hunk 3]
```diff
@@ -7,25 +7,25 @@ require (
...
 	github.com/docker/cli-docs-tool v0.6.0
-	github.com/docker/docker v25.0.0-rc.1+incompatible
+	github.com/docker/docker v25.0.0+incompatible
```
Comment (crazy-max) [inline go.mod:18]: Do we want 25.0.1 bump on buildkit first and revendor here?

Comment (thaJeztah) [reply go.mod:18]: I just opened a PR; https://github.com/moby/buildkit/pull/4591 😅 


I also noticed that docker/cli is using cobra v1.8 (but can do that as a follow-up)

Comment (crazy-max) [reply go.mod:18]: Nice thx!

**Comment by thaJeztah** [comment]
Updated to latest, and update to v25.0.1 (cli and engine); PTAL :heart:

**Review by crazy-max — APPROVED** [review]

**Pull request merged**